examples/src/main/scala/com/gu/tableversions/examples/AdImpressionLoader.scala

```diff
@@ -0,0 +1,63 @@
+package com.gu.tableversions.examples
+
+import java.sql.{Date, Timestamp}
+
+import com.gu.tableversions.examples.AdImpressionLoader.AdImpression
+import org.apache.spark.sql.{Dataset, SaveMode, SparkSession}
+
+/**
+  * This example contains code that writes example event data to a table with multiple partition columns.
+  * It demonstrates how individual partitions in such a table can be updated using versioning.
+  *
+  * @param tableName the fully qualified table name that will be populated by this loader
+  * @param tableUri The location where the table data will be stored
+  */
+class AdImpressionLoader(tableName: String, tableUri: String)(implicit val spark: SparkSession) {
```

Comment:
Wondering if naming these examples quite explicitly might be useful: `TableWithMultiplePartitions`?

Reply:
Yeah I've been through a few iterations of names, it used to be something like that! 😊

But maybe `TableWithMultiplePartitionsLoader` would be better, I can change that.